content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -188,3 +188,55 @@ To resolve this, we suggest you change your deployment to one with a replica set
 One simple way for this is to use [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) to launch a free instance that has replica set support out of the box.
 
 There's also an option to run the replica set locally with this guide: https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set
+
+## Type mapping between MongoDB and the Prisma schema
+
+> Alternatively, see the [Prisma schema reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types) for type mappings organized by Prisma type.
+
+### Mapping from Prisma to MongoDB types on migration
+
+The MongoDB connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) to MongoDB's native column types as follows:
+
+| Prisma     | MongoDB               |
+| ---------- | --------------------- |
+| `String`   | `String`              |
+| `Boolean`  | `Bool`                |
+| `Int`      | `Int`                 |
+| `BigInt`   | `Long`                |
+| `Float`    | `Double`              |
+| `Decimal`  | Currently unsupported |
```

Comment:
Do we have an issue for adding support again when the driver supports it @dpetrick? That would be good for users to complain and also link. (I think we can be pretty open here that MongoDB just does not support that right now and there is not much we can do unfortunately.)

Reply:
Matt created one:
```suggestion
| `Decimal`  | [Currently unsupported](https://github.com/prisma/prisma/issues/12637) |
```